tanner/emulators/sqli.py

```diff
@@ -42,9 +42,13 @@ def map_query(self, attack_value):
     async def get_sqli_result(self, attack_value, attacker_db):
         db_query = self.map_query(attack_value)
         if db_query is None:
-            error_result = 'You have an error in your SQL syntax; check the manual\
-                        that corresponds to your MySQL server version for the\
-                        right syntax to use near {} at line 1'.format(attack_value['id'])
+            if TannerConfig.get('SQLI', 'type') == 'MySQL':
```

Comment:
You made this change [here](https://github.com/mushorg/tanner/pull/336/files)

Reply:
I dont know how it got here. But I have Rebased.